app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
+  def home
```

Comment:
Rubocop will get upset by this, it prefers empty methods to be defined as follows:

```suggestion
  def home; end
```